tools/flye/test-data/result1_assembly_graph.dot

```diff
@@ -0,0 +1,10 @@
+digraph {
+nodesep = 0.5;
```

Comment:
@bgruening this is the assembly graph. Is it a familiar/known format? currently the type is set to `data`

Reply:
I don't think so: https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/datatypes/assembly.py but not sure.

Reply:
https://www.graphviz.org/doc/info/lang.html
How about this? https://github.com/galaxyproject/galaxy/blob/ed6f5d45a9ecbf0cf8a85341e0213b82a9132a46/client/galaxy/scripts/utils/graph.js#L479

Reply:
Yeah, but I haven't seen it here: https://github.com/galaxyproject/galaxy/blob/dev/config/datatypes_conf.xml.sample

Adding a line as `graph_dot` might be a solution. I don't think we can sniff this reliably?

Reply:
I thought, there might be already some vis. plugins for the type in Galaxy. Google brought up this `graphviz - Visualize graph data using cytoscape.js`: https://docs.galaxyproject.org/en/master/slideshow/architecture/galaxy_architecture.html

Reply:
It takes json and a node-* datatype as seen here: https://github.com/galaxyproject/galaxy/blob/dev/config/plugins/visualizations/graphviz/config/graphviz.xml

Reply:
cool, so we should really add a `graph_dot` datatype that subclasses from txt.

Reply:
added a PR for it ...

Reply:
:+1: cool